src/coldfront_plugin_api/scim_v2/groups.py

```diff
@@ -86,6 +89,11 @@ def patch(self, request, pk, format=None):
                     signals.allocation_activate_user.send(
                         sender=self.__class__, allocation_user_pk=au.pk,
                     )
+
+                    self._set_user_on_project(
```

Comment:
Thinking a bit more about this. I would place this above `au = self._set_user_status_on_allocation` as that is the ordering of operations that ColdFront prefers.